appservice/consumers/roomserver.go

```diff
@@ -159,5 +159,33 @@ func (s *OutputRoomEventConsumer) appserviceIsInterestedInEvent(ctx context.Cont
 		}).WithError(err).Errorf("Unable to get aliases for room")
 	}
 
+	// Check if any of the members in the room match the appservice
```

Comment:
I leave this up to the maintainers on the urgency of this, but obviously each event going through dendrite will currently check the membership of a room. If there is an easy way to cache this function call and invalidate it when the membership (or aliases) changes, then let's do that.

Reply:
Opened #1682 to track, we're consciously deferring this work as it's difficult to have caches span several components at once.